packages/debug/src/browser/editor/debug-editor-model.ts

```diff
@@ -222,7 +222,7 @@ export class DebugEditorModel implements Disposable {
                 glyphMarginClassName: className,
                 glyphMarginHoverMessage: message.map(value => ({ value })),
                 stickiness: DebugEditorModel.STICKINESS,
-                beforeContentClassName: typeof column === 'number' ? `theia-debug-breakpoint-column ${className}-column` : undefined
+                beforeContentClassName: typeof column === 'number' ? (column > 1 ? `theia-debug-breakpoint-column ${className}-column` : undefined) : undefined
```

Comment:
please align with https://github.com/microsoft/vscode/blob/355eea22ea9a98a5d0517ffeae5bd176e96bb185/src/vs/workbench/contrib/debug/browser/breakpointEditorContribution.ts#L95

it is the complete solution

Reply:
I've changed the code to mirror the vscode logic. However, I did not find a getLineFirstNonWhitespaceColumn function in the breakpoint class, so instead I assigned a constant and made a comparison against that constant. 

Should I look to implement the getLineFirstNonWhitespaceColumn function instead?

Reply:
this function exists refer to https://github.com/TypeFox/vscode/blob/monaco/0.18.0/src/vs/editor/common/model/textModel.ts#L798 so just use it

Reply:
`this.editor.getControl().getModel()!.getLineFirstNonWhitespaceColumn()`